rubi/rubi/data/helpers/query_types.py

```diff
@@ -29,34 +45,35 @@ def __init__(
         self.open = open
 
     @staticmethod
-    def get_fields(offer_query: Any) -> List:
+    def get_fields(field_paths: Dict[str, FieldPath]) -> List:
```

Comment:
with how we are initializing the `Offer`/`Trade` entities in `_initialize_subgraph_trade` for example, we will most likely want the fields that are returned to have the option to not include the added synthetic fields.